tests/pkg/tests/observability_webhook_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/open-cluster-management/multicluster-observability-operator/tests/pkg/utils"
+)
+
+var (
+	testStorageClassName = "test"
+	testMCOCRName        = "test"
+)
+
+var _ = Describe("Observability:", func() {
+
+	BeforeEach(func() {
+		hubClient = utils.NewKubeClient(
+			testOptions.HubCluster.ClusterServerURL,
+			testOptions.KubeConfig,
+			testOptions.HubCluster.KubeContext)
+	})
+
+	It("[P2][Sev2][Observability][Integration] Create new MCO CR with a storage class that is not allow volume expansion (webhook/g0)", func() {
+		By("Creating a storage class that is not allow volume expansion")
```

Comment:
Just updated to run the tests based on code changes. so you may need to add `webhook/g0` into https://github.com/open-cluster-management/multicluster-observability-operator/blob/cd76fb642b8df7a28f7b3e9ca45b87e20540f44c/cicd-scripts/customize-mco.sh#L117 and https://github.com/open-cluster-management/multicluster-observability-operator/blob/cd76fb642b8df7a28f7b3e9ca45b87e20540f44c/cicd-scripts/customize-mco.sh#L121